### ISSUE TITLE
Suppress linting on iOS?

### ISSUE THREAD

**Issue by michaeleiselsc** [facebook/infer#709]
I see in http://fbinfer.com/support that you can suppress linting with @SuppressLint("infer"), but I was curious is there a way to do this on iOS? For example, I get high false-positive rate from the memory leak check, so I'd like to disable it where necessary.

**Comment by dulmarod** [comment]
No, there isn't currently a way to do @suppresslint("infer") for iOS or suppress specific issues like that, you can suppress classes of issues.

**Comment by michaeleiselsc** [comment]
OK thanks. I'll just add my vote in for this to be a feature. I think I'll have to disable that class of issues, which is unfortunate because it can catch a lot of errors.

**Comment by michaeleiselsc** [comment]
I think I'll solve it for now in a similar way that the clang static analyzer deals with things, by adding -D__infer_enabled__ to each build command and then using #ifdef with that
